.github/workflows/ci.yml

```diff
@@ -4,97 +4,113 @@ on:
   push:
     branches:
       - master
-    tags: [ 'v*' ]
+    tags: ["v*"]
   pull_request:
     branches:
       - master
   schedule:
-    - cron:  '0 6 * * *'  # Daily 6AM UTC build
-
+    - cron: "0 6 * * *" # Daily 6AM UTC build
 
 jobs:
-
   lint:
     name: Linter
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Setup Python
-      uses: actions/setup-python@v2
-    - name: Get pip cache dir
-      id: pip-cache
-      run: |
-        echo "::set-output name=dir::$(pip cache dir)"    # - name: Cache
-    - name: Cache PyPI
-      uses: actions/cache@v2
-      with:
-        key: pip-lint-${{ hashFiles('requirements-dev.txt') }}
-        path: ${{ steps.pip-cache.outputs.dir }}
-        restore-keys: |
-            pip-lint-
-    - name: Install dependencies
-      uses: py-actions/py-dependency-install@v3.0.0
-      with:
-        path: requirements-dev.txt
-    - name: Install itself
-      run: |
-        pip install .
-    - name: Run lint
-      run: |
-        make lint
-    - name: Prepare twine checker
-      run: |
-        pip install -U twine wheel
-        python setup.py sdist bdist_wheel
-    - name: Run twine checker
-      run: |
-        twine check dist/*
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install poetry
+        uses: abatilo/actions-poetry@v2
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+          cache: "poetry"
+      # - name: Setup Python
+      #   uses: actions/setup-python@v2
+      # - name: Get pip cache dir
+      #   id: pip-cache
+      #   run: |
+      #     echo "::set-output name=dir::$(pip cache dir)"    # - name: Cache
```

Comment:
`::set-output::` should be replaced with `$GITHUB_OUTPUT` everywhere

Reply:
I don't believe we need any of the commented-out sections, so I was considering deleting them.